packages/bot-web-ui/src/pages/tutorials/guide-content/__tests__/guide-content.spec.tsx

```diff
@@ -47,21 +53,90 @@ describe('<FAQContent />', () => {
     });
     const mock_DBot_store = mockDBotStore(mock_store, mock_ws);
 
-    beforeEach(() => {
+    beforeAll(() => {
         wrapper = ({ children }: { children: JSX.Element }) => (
             <StoreProvider store={mock_store}>
                 <DBotStoreProvider ws={mock_ws} mock={mock_DBot_store}>
                     {children}
                 </DBotStoreProvider>
             </StoreProvider>
         );
-        render(<GuideContent guide_tab_content={mock_user_guide_content} video_tab_content={mock_guide_content} />, {
-            wrapper,
-        });
     });
 
-    it('should render the component', () => {
+    it('should render the GuideContent component', () => {
+        render(<GuideContent {...mocked_props} />, { wrapper });
+
         expect(screen.getByText('Step-by-step guides')).toBeInTheDocument();
         expect(screen.getByText('Videos on Deriv Bot')).toBeInTheDocument();
     });
+
+    it('should not define the active tour with function the setActiveTour() when it is the desktop version.', async () => {
+        mock_store.ui.is_desktop = true;
+        render(<GuideContent {...mocked_props} />, {
+            wrapper,
+        });
+
+        await waitFor(() => {
+            expect(mock_DBot_store.dashboard.active_tour).toBe('');
+        });
+    });
+
+    it('should open the dialog upon clicking the tutorial video button, should open the tab dashboard upon clicking tutorial card button.', async () => {
```

Comment:
@maryia-matskevich-deriv since these are two different actions can we have two different test cases ?

Reply:
yeah, good idea, you are right, I've split up the test👍